front/src/pods/master/master.component.tsx

```diff
@@ -70,21 +72,40 @@ export const MasterComponent: React.FC<Props> = props => {
   }
 
   return (
-    <div className={classes.container}>
-      <Divider variant="middle" />
-      <Typography className={classes.subtitle} variant="subtitle1">
-        Copy this link to the players that are going to join this poker planning
-        session
-      </Typography>
-      <div className={classes.urlContainer}>
-        <input type="text" value={`${appBaseUrl}/#/player/${room}`} disabled />
-        <FileCopyIcon
-          className={'copyIcon'}
-          onClick={() => {
-            navigator.clipboard.writeText(`${appBaseUrl}/#/player/${room}`);
-          }}
-        />
+    <>
+      <div className={classes.container}>
+        <div className={'left-container'}>
+          <div className={classes.urlContainer}>
+            <Typography className={'subtitle'} variant="subtitle1">
+              Copy this link to the players that are going to join this poker
+              planning session
+            </Typography>
+            <div className={'url'}>
+              <input
+                type="text"
+                value={`${appBaseUrl}/#/player/${room}`}
+                disabled
+              />
+              <FileCopyIcon
+                className={'copyIcon'}
+                onClick={() => {
+                  navigator.clipboard.writeText(
+                    `${appBaseUrl}/#/player/${room}`
+                  );
+                }}
+              />
+            </div>
```

Comment:
Group this into a subcomponent (just in the same file in master.component), name proposal: 

CopySessionLinkComponent